aiohttp/connector.py

```diff
@@ -771,12 +778,18 @@ def _get_ssl_context(self, req):
             sslcontext = req.ssl
             if isinstance(sslcontext, ssl.SSLContext):
                 return sslcontext
+            if not (not isinstance(sslcontext, Fingerprint) or not (sslcontext.ssl_context is not None) or not isinstance(
```

Comment:
There's too many negations, which is hard to understand instantly and easy to misunderstand. I think, this can (and should) be simplified.

Reply:
It seems to me that you want this check:
```python
isinstance(sslcontext, Fingerprint) and isinstance(sslcontext.ssl_context, ssl.SSLContext)
```